src/plugins/OEmbed/Components.tsx

```diff
@@ -92,18 +93,21 @@ export const OEmbed = (
   return <>Unimplemented</>;
 };
 
-// Not sure if we actually want the BlockBaseComponent here,
-// since if the width and height attributes have been set,
-// it will take up that space.
-// If the goal is to just have something that holds the data attributes,
-// it may be better to go back to a plain <article>
-export const OEmbedPlaceholder = (props: BlockBaseProps) => {
+// This Placeholder is only used in renderHTML and is intended to enable
+// data portability/sharing across platform eg. in an RSS feed.
+// Thus we render a plain html article with data-attributes instead of the full BlockBaseComponent (which has some inherent styling)
+// to let the end consumer have full control over how they rebuild the content.
+export const OEmbedPlaceholder = (
+  props: Pick<BlockBaseProps, "attributes" | "pluginName">
+) => {
+  const attributes = props.attributes;
+  const dataAttributes = makeDataAttributes(attributes);
   return (
-    <BlockBaseComponent
-      {...props}
+    <article
       className="embed-placeholder"
```

Comment:
Do we need the class name here? i feel like we should avoid classes in the data transfer things (but I'm out of brain, so forgive me if this is obvious)

Reply:
No, you're right, I just didn't think to take it out once we figured out what we were actually trying to do here